app/src/main/java/com/example/dogbook/main/MainActivity.java

```diff
@@ -42,9 +43,23 @@ private void initView() {
         navigationBar.setOnNavigationItemSelectedListener(new BottomNavigationView.OnNavigationItemSelectedListener() {
             @Override
             public boolean onNavigationItemSelected(MenuItem item) {
-                Fragment fragment = new TimelineFragment();
-                fragmentManager.beginTransaction().replace(R.id.fragmentContainer, fragment).commit();
-                return true;
+                Fragment fragment;
+                switch (item.getItemId()) {
+                    case R.id.item_home:
+                        fragment = new TimelineFragment();
+                        fragmentManager.beginTransaction().replace(R.id.fragmentContainer, fragment).commit();
+                        return true;
+
+                    case R.id.item_dogs_nearby:
+                        fragment = new MapFragment();
+                        fragmentManager.beginTransaction().replace(R.id.fragmentContainer, fragment).commit();
+                        return true;
+
+                    default:
+                        fragment = new TimelineFragment();
+                        fragmentManager.beginTransaction().replace(R.id.fragmentContainer, fragment).commit();
+                        return true;
```

Comment:
You can merge the timeline and default case here that way you don't have duplicate code:

```
case R.id.item_home:
default:
                        fragment = new TimelineFragment();
                        fragmentManager.beginTransaction().replace(R.id.fragmentContainer, fragment).commit();
                        return true;
```